adafruit_ltr390.py

```diff
@@ -91,7 +102,10 @@ class CV:
     """struct helper"""
 
     @classmethod
-    def add_values(cls, value_tuples):
+    def add_values(
+        cls,
+        value_tuples: Tuple[str, int, str, Union[int, None], int, Union[float, None]],
```

Comment:
For `Union[X, None]`, you should use `Optional[X]`.  Also I think `lsb` should be `Optional[float]` since it represents what an LSB can represent, which is based on this design pattern in another libraries.

Reply:
Additionally, `value_tuples` should be wrapped in `Iterable` (imported from `typing`) based on the way it's accessed:

```python
Iterable[Tuple[str, int, str, Optional[float], int, Optional[float]]]
```